packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -85,9 +81,7 @@ export function registerLocalNetwork() {
   try {
     const customL1Network = localNetwork.l1Network
     const customL2Network = localNetwork.l2Network
-
-    rpcURLs[customL1Network.chainID] = customL1Network.rpcURL
-    rpcURLs[customL2Network.chainID] = customL2Network.rpcURL
+    // TODO: Remove this?
```

Comment:
Let's not remove this, it's handy for registering a local network

Reply:
how do we do this without rpcURL? it doesn't exist on the network object anymore